warp-tls/Network/Wai/Handler/WarpTLS.hs

```diff
@@ -182,6 +182,7 @@ tlsSettingsSni onServerNameIndicationHook = do
 -- representations of the certificate and key based on 'defaultTlsSettings'.
 --
 -- @since 3.3.0
+{-# DEPRECATED tlsSettingsRef "use tlsSettingsChainIORef, which avoids a race condition and receives dynamic updates" #-}
```

Comment:
Where's `tlsSettingsChainIORef`?

Reply:
Those were not meant to be there. See #1024 for the correct deprecations and #859 for context.